.pre-commit-config.yaml

```diff
@@ -5,4 +5,17 @@ repos:
     rev: v3.2.0
     hooks:
     -   id: trailing-whitespace
+    -   id: mixed-line-ending
     -   id: end-of-file-fixer
+    -   id: debug-statements
+-   repo: https://github.com/pycqa/flake8
+    rev: 3.9.2
+    hooks:
+    -   id: flake8
+        name: "Flake8: critical"
+        args: ['--count', '--select=E9,F63,F7,F82', '--show-source', '--statistics']
+-   repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+    -   id: black
+        exclude: migrations/
```

Comment:
Ticket-plugins will never have migrations. Then they would be apps and would have to be put into INSTALLED_APPS as well.